dotnet-desktop-guide/framework/winforms/controls/defining-default-values-with-the-shouldserialize-and-reset-methods.md

```diff
@@ -21,16 +21,20 @@ ms.assetid: 7b6c5e00-3771-46b4-9142-5a80d5864a5e
     > [!NOTE]
     > Either apply the <xref:System.ComponentModel.DefaultValueAttribute> or provide `Reset`*PropertyName* and `ShouldSerialize`*PropertyName* methods. Do not use both.
 
+    > [!NOTE]
+    > These methods are invoked by the designer by convention, and as such can be made `private`.
```

Comment:
We shouldn't have back-to-back alerts. It causes people to skip over them. Maybe just state it? This suggestion below fixes the indentation. I'm unsure why the first note is part of the "designer generates more efficient code" bullet point.

How does this sound?

```suggestion
> [!NOTE]
> Either apply the <xref:System.ComponentModel.DefaultValueAttribute> or provide `Reset`*PropertyName* and `ShouldSerialize`*PropertyName* methods. Do not use both.

When declaring a `ShouldSerialize` or `Reset` method, use the `private` access modifier. These methods are usually invoked by the designer and not by user code.
```

Reply:
Sounds good! Thank you